swift_scality_backend/diskfile.py

```diff
@@ -277,6 +295,31 @@ def __repr__(self):
         return ret % (self._client_collection, self._account, self._container,
                       self._obj, self._use_splice, self._logger)
 
+    @staticmethod
+    def merge_df_mf_metadata(df_md_source, mf_md_source):
+        """Merge the datafile metadata and metafile metadata dictionaries together.
+
+        Datafile metadata refers to metadata originally included when the object
+        was first PUT, and does not include metadata set by any subsequent POST.
+
+        Metafile metadata refers to metadata written by a POST, and does not
+        include any persistent metadata that was set by the original PUT.
+        """
+        md_dest = {}
+
+        if not mf_md_source:
+            md_dest.update(df_md_source)
+        else:
+            sys_metadata = {
```

Comment:
This breaks Python 2.6 compatibility (but maybe we no longer support that anyway?)

Reply:
Yes, swift has a dependency on Python 2.7.
Also, the support of Python 2.6 was disabled in [this commit](https://github.com/scality/ScalitySproxydSwift/commit/0e03197de25c3b0e35267e561b69365260684caf)